src/shared/ui/survey/TextParagraph.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC, useState } from 'react';
+import {
+  StyleSheet,
+  TextInputProps,
+  View,
+  NativeSyntheticEvent,
+  TextInputContentSizeChangeEventData,
+} from 'react-native';
+
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@shared/lib';
+import { LongTextInput } from '@shared/ui';
+
+type Props = {
+  onChange: (text: string) => void;
+  value: string;
+  config: {
+    maxLength: number;
+    isNumeric: boolean;
+  };
+} & Omit<TextInputProps, 'value' | 'onChange'>;
+
+const TextParagraph: FC<Props> = ({ value, onChange, config, ...props }) => {
+  const { maxLength = 50, isNumeric } = config;
+  const { t } = useTranslation();
+
+  const [inputHeight, setInputHeight] = useState(56);
+
+  const onChangeText = (text: string) => {
+    onChange(text);
+  };
+
+  const handleContentSizeChange = (
+    contentWidth: number,
+    contentHeight: number,
+  ) => {
+    setInputHeight(contentHeight);
+  };
+
+  const handleContentSizeChangeWrapper = (
+    event: NativeSyntheticEvent<
+      | {
+          contentWidth: number;
+          contentHeight: number;
+        }
```

Comment:
According to the React Native's documentation: https://reactnative.dev/docs/textinput#oncontentsizechange
There is a defined interface for the event object: `({nativeEvent: {contentSize: {width, height} }}) => void`

Why would we need to have another one like the one you described above?